hack/install-deny-all-net-pol.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex
+#
+# Copyright 2025 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script install deny-all NetworkPolicy that affects kubevirt-ipam-controller namespace.
+
+readonly ns="$(${KUBECTL} get pod -l app=ipam-virt-workloads -A -o=custom-columns=NS:.metadata.namespace --no-headers | head -1)"
+[[ -z "${ns}" ]] && echo "FATAL: kubevirt-ipam-controller pods not found. Make sure kubevirt-ipam-controller is installed" && exit 1
+
+readonly np_name="deny-all"
+${KUBECTL} -n "${ns}" get networkpolicy "${np_name}" ||
+  cat <<EOF | ${KUBECTL} -n "${ns}" create -f -
```

Comment:
let's make this idempotent by using apply instead. 

I think this'll fail if sync is invoked twice.

Reply:
You totally right, it misses some bits that makes it idempotent.

Reply:
I verified it works when sync invoked multiple times, missing part was passing stdout/err to /dev/null.

Reply:
Its possible, but I dont think we need to use apply because this policy is not expected to changed.
Note that it wont create the object if its already exist.

Reply:
ok, then it's just my OCD at play ... still I got to ask, why use create vs apply ?

Reply:
I dont have strong opinions in this particular case. I changed it to use apply, PTAL.